sumpy/fmm.py

```diff
@@ -736,18 +746,19 @@ def multipole_to_local(self,
             target_boxes, src_box_starts, src_box_lists,
             mpole_exps):
 
-        precompute_evts = []
+        preprocess_evts = []
         queue = mpole_exps.queue
+        local_exps = self.local_expansion_zeros(mpole_exps)
 
-        if self.use_preprocessing_for_m2l:
+        if self.supports_optimized_m2l:
```

Comment:
More concrete naming for this flag? (I'd already forgotten what it's for.)

Reply:
Done in 52044e195422f357b03158f15187681ff6a80617